content/news/2024_03_18_GRASS_Community_meeting_Prague_2024.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "2024 GRASS GIS Community meeting in Prague!"
+date: 2024-03-18T14:12:42-05:00
+layout: "news"
+author: Veronica Andreo and the GRASS Development Team
+---
+
+### The GRASS GIS team announces the annual **Community Meeting**!! 
+
+The [GRASS GIS Community Meeting](https://grasswiki.osgeo.org/wiki/GRASS_Community_Meeting_Prague_2024) 
+will take place from **June 14 to 19, 2024**, at the 
+[NC State European Center in Prague](https://prague.ncsu.edu/about/), Czech Republic.
+
+Community meetings are great opportunities to support the development of GRASS GIS!
+
+<a href="/images/news/grass_community2023_prague_fotowall.jpg">
+  <img src="/images/news/grass_community2023_prague_fotowall.jpg"
+   alt="Prague 2023 Community Meeting"
+   title="Prague 2023 Community Meeting"
+   width="55%" style="float:right;padding-left:25px;padding-top:15px">
+</a>
+
+#### Join us
+
+* Write code
+* Write documentation
+* Translation
+* Website design and content
+* Integrations with other software (QGIS, GDAL, R, etc.)
+
+#### Plan for the future
+
+The community meeting is a get-together where **supporters**, **contributors**, 
+**power users** and **developers** make decisions and tackle larger problems 
+related to the project, discuss and fix bugs, draw the project roadmap and 
+work on new features. 
+
+#### Let's celebrate!
+
+Join us in celebrating **the 41st anniversary of GRASS GIS** and the community's 
+ongoing dedication and achievements over the past year.
+
+#### Support us
+
+To support as many attendants as possible **we welcome sponsorship and financial support** 
+via [OpenCollective](https://opencollective.com/osgeo/projects/grass/contribute).
+Your contribution makes a difference!!
+
+<div align="center">
+<button class="btn btn-primary">
```

Comment:
Maybe with btn-outline the contrast would be better? I don't know what version of bootstrap it is.
```suggestion
<button class="btn btn-outline-primary">
```


There's also normally the bootstrap class `btn-lg` to make it bigger, if it is supported by your version. 


For the text of the link, there are classes for it, like `text-white`, but for this class specifically it might cancel out styling of the link once clicked.

I assumed you weren't on bootstrap 5.3 (latest), so some info that might apply is here (for version 4.0):

https://getbootstrap.com/docs/4.0/utilities/colors/

https://getbootstrap.com/docs/4.0/components/buttons/

Reply:
That did not work, we created a new button class